profile/README.md

```diff
@@ -47,13 +47,20 @@ We have a few actively developing public reference products:
 - Disabling/enabling L3 offload via sysfs file
 - PCIe SR-IOV support
 
+[meta-xt-prod-devel-rpi5](https://github.com/xen-troops/meta-xt-prod-devel-rpi5) for Raspbery Pi 5
+- Zephyr operated control domain
+- Support dom0less functionality
+- Linux operated driver domain
+- PV hardware backends
+- Linux, Unikraft or Zephyr as guest domain
```

Comment:
+ OP-TEE support

I am wondering if we should mention limitations as well:  what immediately comes to min is "no IOMMU" and "no secure memory"

Reply:
`OP-TEE support` is added,

I'm not sure we need to add a lot of text here. It could be better to provide details and limitations in the README file of the corresponding product.
Even more, I think we should rework this README to make it shorter. But I'm not sure now.